front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -187,14 +203,31 @@ const SimulationResultsV2 = ({
               pathProperties={pathProperties}
               sharedXScaleDomain={positionScaleDomain}
               setSharedXScaleDomain={setPositionScaleDomain}
-              departureTime={selectedTrain.start_time}
+              departureTime={selectedTrainSchedule.start_time}
             />
           </div>
         </div>
       )}
 
+      {selectedTrainSchedule &&
```

Comment:
This section doesn't have a `{/* */}` comment before like the other ones on the page

Reply:
done